packages/landing/src/widgets/landing/subscription-section.tsx

```diff
@@ -50,7 +51,7 @@ export function SubscriptionSection() {
           <img src="/img/email.svg" alt="" className="email" />
         </Form>
       ) : (
-        <Form>
+        <Form onSubmit={(e: any) => handleSubscribe(e)}>
```

Comment:
`any` isn't a good solution of TypeScript problems.

We can remove type of `e` and call `e.preventDefault()` right here. Or we can use `FormEvent<HTMLFormElement>` as it's inferred automatically.